README.md

```diff
@@ -37,6 +37,7 @@ UffDbLoader.configure do |config|
   config.app_name = 'my_app' # Defaults to the Rails app name
   config.dumps_directory = '/path/to/dumps' # Defaults to Rails.root.join('dumps')
   config.database_config_file = 'path/to/database.yml' # Defaults to Rails.root.join('config', 'database.yml')
+  config.container_name_fn = ->(app_name, environment) { "#{app_name}_#{environment}_custom_suffix" } # Defaults to "#{app_name}_#{environment}_db".
```

Comment:
Should we add this to the initializer created by our install script?

Reply:
Oh! Yes!